src/org/openstreetmap/josm/plugins/osmobjinfo/OSMObjInfoActions.java

```diff
@@ -61,29 +74,93 @@ public static void openinBrowserChangeset(String idChangeset) {
+        clipboard.setContents(selection, selection);
+        new Notification(tr(COPY, linkobjid)).setIcon(JOptionPane.INFORMATION_MESSAGE).setDuration(Notification.TIME_SHORT).show();
+    }
+
+    public static void copyRemoteControl(List<AllOsmObjInfo> osmObjInfo) {
+        if (osmObjInfo == null)
+            return;
+        StringBuilder builder = new StringBuilder();
+        try {
+            builder.append("http://");
+            builder.append(RemoteControl.getInet4Address().getHostAddress());
+            builder.append(":");
+            builder.append(Config.getPref().getInt("remote.control.port", 8111));
+        } catch (UnknownHostException e) {
+            // Append the default remote control address (this gets thrown at RemoteControl#getInet4Address)
+            builder.append("127.0.0.1:8111");
+        }
+        builder.append("/").append(LoadAndZoomHandler.command).append("?");
+        if (!osmObjInfo.isEmpty())
+            builder.append("select=");
+        for (int i = 0; i < osmObjInfo.size(); i++) {
+            AllOsmObjInfo object = osmObjInfo.get(i);
+            builder.append(object.typeObj);
+            builder.append(object.idObject);
+            if (i < osmObjInfo.size() - 1)
+                builder.append(',');
+        }
+        Bounds viewPortBound = MainApplication.getMap().mapView
+                .getLatLonBounds(MainApplication.getMap().mapView.getVisibleRect());
+        // We don't need to check that the length is > 0, as we have the remote control URL
+        if (builder.charAt(builder.length() - 1) != '?') {
+            builder.append('&');
         }
+        builder.append("left=").append(viewPortBound.getMinLon()).append("&right=").append(viewPortBound.getMaxLon())
+                .append("&top=").append(viewPortBound.getMaxLat()).append("&bottom=").append(viewPortBound.getMinLat());
+        String remoteControl = builder.toString();
+        StringSelection selection = new StringSelection(remoteControl);
+        Clipboard clipboard = Toolkit.getDefaultToolkit().getSystemClipboard();
+        clipboard.setContents(selection, selection);
+        new Notification(tr(COPY, remoteControl)).setIcon(JOptionPane.INFORMATION_MESSAGE)
+          .setDuration(Notification.TIME_SHORT).show();
     }
 
-    public static void openinBrowserIdobj(String typeObj, String idobj) {
-        if (typeObj != null && !idobj.isEmpty()) {
-            openInBrowser("https://www.openstreetmap.org/" + typeObj + "/" + idobj);
+    public static void openinBrowserIdobj(List<AllOsmObjInfo> osmObjInfo) {
+        if (osmObjInfo.size() > OSMObjInfoPlugin.MAXIMUM_SELECTION.get() || osmObjInfo.isEmpty())
+            return;
+        for (AllOsmObjInfo info : osmObjInfo) {
+            if (info.typeObj != null && !info.idObject.isEmpty()) {
+                openInBrowser("https://www.openstreetmap.org/" + info.typeObj + "/" + info.idObject);
```

Comment:
Config.getUrls().getBaseBrowseUrl() should be used instead of hardcoding https://www.openstreetmap.org